conseil-common/src/test/scala/tech/cryptonomic/conseil/common/tezos/TezosTypesTest.scala

```diff
@@ -3,43 +3,42 @@ package tech.cryptonomic.conseil.common.tezos
 import java.time.Instant
 
 import org.scalatest.{EitherValues, Matchers, OptionValues, WordSpec}
+import tech.cryptonomic.conseil.common.generic.chain.DataTypes.BlockHash
 import tech.cryptonomic.conseil.common.tezos.TezosTypes._
 
 class TezosTypesTest extends WordSpec with Matchers with OptionValues with EitherValues {
 
-  val sut = TezosTypes
-
   "The Base58Check verifier" should {
       "accept an empty string" in {
-        sut.isBase58Check("") shouldBe true
+        TezosTypes.isBase58Check("") shouldBe true
```

Comment:
was that so bad, that it needed to be changed? 😅

Reply:
Well, no, but that was confusing for me, that we have `val` for an `object`. I even forgot that I did this change here. I did that when `BlockHash` has been moved to different location :D

Reply:
Well, it's a weird an unusual choice, I only made it because I liked that test could clearly state which is the thing we're testing.
That is not so apparent when you use an object as a test-subject 😁

Reply:
I see, when you are looking at single test-case, and you see only one line there, I think that's pretty clear. I would like it more, once we would have more methods in the "test-case scope", which can blur what we are testing, but I'm like a guest here. If you want it, I can revert it back ;)

Reply:
I honestly am always in a hard spot in this kind of choices.
I use this "convention" but it's not a shared thing, so it's not even enforced in the codebase overall...
Would this really make a difference? I only share my PoV for the rest of the team to have feedback, 'cause I hope to build a common approach.
But it's kind of tiring to uphold to such kind of "methodology" if there seems to be poor interest in this kind of subtleties. If my choice is not shared, I'm driven to give up, since it's probably not relevant.

I'm not here to enforce a code style, I only want to target what we can commonly consider good design choices.

Reply:
@piotrkosecki can you also give feedback on that topic. You have decisive one ^^

Reply:
Both arguments are reasonable, but I would probably keep the name `sut` - it follows the pattern we use in most of our tests.